run_tests.sh

```diff
@@ -14,7 +14,11 @@ export NITRO_CLI_BLOBS="${SCRIPTDIR}/blobs"
 export NITRO_CLI_ARTIFACTS="${SCRIPTDIR}/build"
 ARCH="$(uname -m)"
 
-$(aws ecr get-login --no-include-email --region us-east-1)
+AWS_ACCOUNT_ID=667861386598
```

Comment:
nit: Could these variables be read from an external file and then exported?

Reply:
Potentially, yes. But what would be the benefit?

Reply:
For better maintainability. We could combine (not only the variables above but) all hard-coded values in a file and source it from any of the scripts.

Reply:
I also see some hard-coded docker registry addresses (667861386598.dkr.ecr.us-east-1.amazonaws.com) in this file that could be replaced by $ECR_URL.

Reply:
Makes sense, but I'd like to do that in another patch series. I'd actually like to try and drop the private ECR containers as dependencies, so these tests can just be run by anyone from anywhere.